Dangerfile.js

```diff
@@ -1 +1,23 @@
-message("Hello, world.")
+// CHANGELOG check
+if (_.includes(git.modified_files, "CHANGELOG.md") === false) {
+  fail("No CHANGELOG added.")
+}
+
+const test_files = _.filter(git.modified_files, function(path){
```

Comment:
Nice! I would observe that in my rush to give you some good sample code I wasn't necessarily following good style. Should these `const` names all be camelcase instead of snakecase?

And is that the sort of thing Danger would complain about? 😉